src/_locales/ja/messages.json

```diff
@@ -467,9 +467,6 @@
   "settings_extract_pronouns": {
     "message": "ユーザーの自己紹介や位置情報から代名詞を抽出し、カラムに表示する"
   },
-  "settings_mute_nfts_accounts": {
-    "message": "NFTアイコンを使用しているユーザーをミュートする(六角形のアイコン)"
-  },
```

Comment:
This translation seems to have been removed unexpectedly.

Reply:
Feel free to re-add it into Crowdin, re-adding from GitHub can lead to weird conflicts 😓